source/reference/minio-mc-admin/mc-admin-accesskey-create.rst

```diff
@@ -0,0 +1,232 @@
+
+.. end-mc-admin-accesskey-create-desc
+
+.. admonition:: Access keys for OpenID Connect or AD/LDAP users
+   :class: note
+
+   This command is for access keys for users created directly on the MinIO deployment and not managed by a third party solution.
+
+   - To generate access keys for :ref:`OpenID Connect users <minio-external-identity-management-openid>`, use the :ref:`MinIO Console <minio-console>`.
+
+   - To generate access keys for :ref:`Active Directory/LDAP users <minio-external-identity-management-ad-ldap>`, use :mc:`mc idp ldap accesskey create`.
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command creates a new access key associated to an existing MinIO user:
+
+      .. code-block:: shell                                                                 
+         :class: copyable
+
+         mc admin accesskey create        \
+            myminio/ myuser               \                                                  
+            --access-key myuseraccesskey  \                                  
+            --secret-key myusersecretkey  \
+            --policy /path/to/policy.json 
+                                                                                   
+      The command returns the access key and secret key for the new account.
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc [GLOBALFLAGS] admin accesskey create                    \
+                                          ALIAS                     \
+                                          [USER]                    \
+                                          [--access-key string]     \
+                                          [--secret-key string]     \
+                                          [--policy path]           \
+                                          [--name string]           \
+                                          [--description string]    \
+                                          [--expiry-duration value] \
+                                          [--expiry date]
+					
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: ALIAS
+   :required:
+
+   The :mc-cmd:`alias <mc alias>` of the MinIO deployment.
+
+.. mc-cmd:: USER
+   :optional:
+
+   The username of the user to which MinIO adds the new access key.
+   If not specified, MinIO generates an access key/secret key pair for the authenticated user.
+
+.. mc-cmd:: --access-key
+   :optional:
+
+   A string to use as the access key for this account.
+   Omit to let MinIO autogenerate a random 20 character value.
+
+   Access Key names *must* be unique across all users.
+
+.. mc-cmd:: --description
+   :optional:
+
+   Add a description for the service account.
```

Comment:
But the access key belongs to the user, right? So how is the service account a part of the story with the new command? Is this left over from the older `svcacct` command?

Reply:
That should definitely be "access key" there and the next line.

Reply:
Doing a pass throughout to catch them all.